### DISCUSSION TITLE
Extra options for shared character slots

### DISCUSSION THREAD

**Discussion by smackyjackson** [ikemen-engine/Ikemen-GO#1424] [Feature Request]
### Describe the solution you'd like

I had two ideas for characters that share the same select slot. I doubt if either of these are remotely possible, but I thought I'd throw it out there.
The first was the option to tag exclude on a slotted character, and still allow us to select them manually via the command. Essentially a work around to get them excluded from randomselect but still be selectable.
The other is more commands to use on slotted characters. It would be really cool to be able to select slotted characters via complex button inputs. Something akin to old school secret characters. Like being able to highlight a character, and inputting the Konami code to get a secret character.
Maybe allowing for directional inputs for directions that are not next to a selectable spot. So if its in the top right corner, you could do up, up, right, right, up, A, B, start, and get something.

### Search terms

Character slots, Select.def, Hidden characters

**Comment by ZyonZZ** [comment]
or how to unlock Kusanagi in Kof 2002. by moving the lever "Up, Down, Right, Left, Down, Up, Left, Right" and change the character portrait.

Thanks to the creator of ikemen you have the option of modules or Mods... will it be possible to incorporate something like this through Mods?
It's also titanic to ask so many functions from a creator when the community should also create functions...

mods are released daily for Skyrim, Minecraft, 7days ETC... and ikemen No... I would like to know how to program to help :(

There are projects like I.K.E.M.E.N. PLUS ULTRA that they do apply improvements... but since they are not through mods but rather they modify the ENTIRE BASE OF THE ENGINE and add tons of zzs, lua and other codes... it becomes complicated, it is almost impossible to use screenpacks etc due to engine incompatibilities modified....
Technically it is only for programmers... something that makes no sense to me...
